src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -148,14 +178,18 @@ class SubjectHeading extends React.Component {
     } = this.addEmphasis(label);
 
     return (
-      <li data={`${subjectHeading.uuid}, ${container}`}>
-        <a className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""}` + `${this.props.nested ? ' nestedSubjectHeading' : ''}`} >
+      <li data={`${subjectHeading.uuid}, ${container}`} className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""}`}>
+        <a  className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""} + ${this.props.nested ? ' nestedSubjectHeading' : ''}`} >
```

Comment:
do the class names need to be on both? Also, how is the `data` property on the `li` working?

Reply:
The data property is just a convenience to be able to see the uuid using developer tools. As for the extra styling on the a tag, I added it back for some reason but now I don't see any problems without it, so I'm removing it.

Reply:
Actually the extra styling is to get the attribute values aligned to the right. I am reverting back to adding that styling

Reply:
Why not just make a selectors like `.subjectHeadingRow.openSubject > a`?